example.diff

```diff
@@ -1,3 +1,52 @@
+--- a/document.txt
++++ b/document.txt
+@@ -1,5 +1,5 @@
+-The quick brown fox jumps over the lazy dog.
++The fast brown fox leaps over the sleeping dog.
+ She sells seashells by the seashore.
+-The rain in Spain stays mainly in the plain.
++The rain in France falls mainly on the plain.
```

Comment:
This example diff shows a bug:
![2024-07-22-15-32](https://github.com/user-attachments/assets/0d52d135-53ea-4651-8845-e692add2bd67)

I don't know what causes this but I think it's probably related to the lack of separation between the code that computes the word diffs from the code that output them.